graph/three_edge_connected_components/info.toml

```
@@ -0,0 +1,32 @@
+title = 'Three Edge Connected Components'
```

```suggestion
title = 'Three-Edge-Connected Components'
```
google様に伺うと全部ハイフンがついていそうな感じがします(英弱)